src/java.base/unix/classes/sun/nio/fs/UnixFileAttributes.java

```diff
@@ -81,8 +81,11 @@ static UnixFileAttributes get(UnixPath path, boolean followLinks)
         return attrs;
     }
 
-    // get the UnixFileAttributes for a given file. Returns null if the file does not exist.
-    static UnixFileAttributes getIfExists(UnixPath path) throws UnixException {
+    // get the UnixFileAttributes for a given file.
+    // Returns null if the file does not exist.
+    static UnixFileAttributes getIfExists(UnixPath path)
+        throws UnixException
+    {
```

Comment:
It would be nice if getIfExists(path) could be implemented as getIfExists(path, true) but it can't because the former doesn't need an exception when the file doesn't exists where the latter does. Something to look into sometime.